src/main/java/com/xiao/users/service/IUserService.java

```diff
@@ -14,4 +14,6 @@ public interface IUserService {
 
     UserDto updateUser(Long userId, UserUpdateDto userDto);
 
+    void deleteUser(Long userId);
```

Comment:
Why is the return type of the REST controller method `ResponseEntity<UserDto>`, but the return type of deleteUser in the service is `void`?